src/acachecontrol/acachecontrol.py

```diff
@@ -27,6 +32,15 @@ def get(self, url, *, allow_redirects=True, **params):
     def post(self, url, *, data, **params):
         return self.request("POST", url, data=data, **params)
 
+    def put(self, url, *, data, **params):
+        return self.request("PUT", url, data=data, **params)
+
+    def patch(self, url, *, data, **params):
```

Comment:
I don't think that we need `*` in methods arguments.
```suggestion
    def patch(self, url, data, **params):
```
It is applicable to other methods.

Reply:
I used it from original aiohttp as it invokes aiohttp methods. Seems legit.